ccd-definition/CaseEvent/User/UserEvents-SDO-nonprod.json

```diff
@@ -15,5 +15,23 @@
     "EndButtonLabel": "Submit",
     "RetriesTimeoutAboutToStartEvent": 0,
     "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "REFER_TO_JUDGE",
+    "Name": "Refer To Judge",
+    "Description": "Refer To Judge",
+    "DisplayOrder": 18,
+    "PreConditionState(s)": "JUDICIAL_REFERRAL",
+    "PostConditionState": "CASE_PROGRESSION",
```

Comment:
Does the case not stay in the 'Judicial Referral' state once it has been referred to a judge?